test/e2e/storage/csimock/csi_honor_pv_reclaim_policy.go

```diff
@@ -27,15 +27,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	storagehelpers "k8s.io/component-helpers/storage/volume"
 	"k8s.io/kubernetes/pkg/features"
-	"k8s.io/kubernetes/test/e2e/feature"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
 	"k8s.io/kubernetes/test/e2e/storage/utils"
 	admissionapi "k8s.io/pod-security-admission/api"
 	"k8s.io/utils/ptr"
 )
 
-var _ = utils.SIGDescribe("CSI Mock honor pv reclaim policy", feature.HonorPVReclaimPolicy, framework.WithFeatureGate(features.HonorPVReclaimPolicy), func() {
+var _ = utils.SIGDescribe("CSI Mock honor pv reclaim policy", framework.WithFeatureGate(features.HonorPVReclaimPolicy), func() {
```

Comment:
We cannot do that yet. Any job which filters with `--skip="Feature:.*"` would start to run this test. We first have to be sure that "all" jobs have been converted to using `--label-filter`.